pkg/protocol/xprotocol/boltv2/protocol.go

```diff
@@ -125,22 +125,47 @@ func (proto *boltv2Protocol) Decode(ctx context.Context, data types.IoBuffer) (i
 }
 
 // heartbeater
-// boltv2 send bolt heartbeat
+// set version1 as default 0x01, no crc, use version 1
+// see details in: https://github.com/sofastack/sofa-bolt/blob/master/src/main/java/com/alipay/remoting/rpc/protocol/RpcCommandEncoderV2.java
 func (proto *boltv2Protocol) Trigger(ctx context.Context, requestId uint64) api.XFrame {
-	engine := xprotocol.GetProtocol(bolt.ProtocolName)
-	return engine.Trigger(ctx, requestId)
+	return &Request{
+		RequestHeader: RequestHeader{
+			Version1: 0x01,
```

Comment:
`0x01` appears in multiple places, so it might be better to use a constant variable.